l1-contracts/contracts/state-transition/IChainTypeManager.sol

```diff
@@ -59,6 +59,9 @@ interface IChainTypeManager {
     /// @notice ValidatorTimelock changed
     event NewValidatorTimelock(address indexed oldValidatorTimelock, address indexed newValidatorTimelock);
 
+    /// @notice ServerNotifier changed
+    event NewServerNotifier(address indexed oldServerNotifier, address indexed newServerNotifier);
```

Comment:
nit: btw I doubt that we need `Indexed` here, but nevermind

Reply:
I took the approach from `NewValidatorTimelock`